apps/admin-portal/src/app.tsx

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
This file was added last year so we can keep the same license header I guess.

Reply:
Fixed with commit `48590b7`